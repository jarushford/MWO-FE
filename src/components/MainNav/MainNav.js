import React, { useState } from 'react'
import { NavLink, Link } from 'react-router-dom'
import '../../main.scss'

export default function MainNav() {
  const [menu, setMenu] = useState(false)

  return (
    <section>
      <header className="main-nav nav">
        <NavLink to="/tour">tour</NavLink>
        <NavLink to="/media">media</NavLink>
        <Link to="/">
          <div className="header-logo">
            <img alt="mw-logo" src="./assets/main-logo.png" />
          </div>
        </Link>  
        <NavLink to="/about">about</NavLink>
        <NavLink to="/contact">contact</NavLink>
      </header>
      <header className="mobile-nav nav">
        <Link to="/" onClick={() => setMenu(false)}>
          <div className="header-logo">
            <img alt="mw-logo" src="./assets/main-logo.png" />
          </div>
        </Link>
        <div
          className={`burger ${menu && 'open'}`}
          onClick={() => setMenu(!menu)}
        >
          <div className="bar1" />
          <div className="bar2" />
          <div className="bar3" />
        </div>
        <div className={`dropdown-container ${menu && 'show'}`}>
          <NavLink
            onClick={() => setMenu(false)}
            to="/tour"
          >
            tour
          </NavLink>
          <NavLink
            onClick={() => setMenu(false)}
            to="/media"
          >
            media
          </NavLink>
          <NavLink
            onClick={() => setMenu(false)}
            to="/about"
          >
            about
          </NavLink>
          <NavLink
            onClick={() => setMenu(false)}
            to="/contact"
          >
            contact
          </NavLink>
        </div>  
      </header>
    </section>
  )
}