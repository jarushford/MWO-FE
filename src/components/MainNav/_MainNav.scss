.nav {
  position: fixed;
  display: flex;
  justify-content: space-around;
  align-items: center;
  background: linear-gradient(to right, rgba(47,47,47,.8) , rgba(47,47,47,.9), rgba(47,47,47,1), #3f3f3f, rgba(47,47,47,1), rgba(47,47,47,.9), rgba(47,47,47,.8));
  border-bottom: 3px solid #b999c8;
  position: fixed;
  z-index: 101;
  top: 0px;
  left: 0px;
  right: 0px;
  height: 75px;

  a,
  a:visited {
    color: white;
    text-decoration: none;
    font-size: 2rem;
    font-family: 'Open Sans', sans-serif;
    height: 80px;
    width: 100%;
    text-align: center;
    line-height: 75px;
    transition: .3s ease;
    position: relative;

    &:nth-child(3) {
      height: 0;

      &:hover::after,
      &:hover::before {
        width: 0%;
      }
    }
    
    &::before {
      content: '';
      position: absolute;
      top: 15px;
      left: 0;
      right: 0;
      margin: 0 auto;
      height: 2px;
      width: 0%;
      background: #b45dd6;
      transition: .3s ease;
    }

    &::after {
      content: '';
      position: absolute;
      bottom: 15px;
      left: 0;
      right: 0;
      margin: 0 auto;
      height: 2px;
      width: 0%;
      background: #b45dd6;
      transition: .3s ease;
    }

    &:hover {
      color: #b45dd6;
    }

    &:hover::after,
    &:hover::before {
      width: 50%;
    }
  }

  a.active {
  color: #b45dd6;

    &::after,
    &::before {
      width: 30%;
    }
  }

  img {
    width: 80px;
    margin-right: 24px;
  }
}

.mobile-nav {
  display: none;

  img {
    width: 50px;
    margin-left: 15px;
  }

  .header-logo {
    width: 100px;
    height: 100px;
    max-width: 100px;
    min-width: 100px;
    position: absolute;
    top: 0;
    left: 0;
  }

  a,
  a:visited {

    &:nth-child(3) {
      height: 80px;
    }
  }
}

.dropdown-container {
  display: flex;
  flex-direction: column;
  float: right;
  margin-top: 78px;
  padding: 20px 30px;
  background: linear-gradient(to top, rgba(47,47,47,.8) , rgba(47,47,47,.9), rgba(47,47,47,1), #3f3f3f, rgba(47,47,47,1), rgba(47,47,47,.9), rgba(47,47,47,.8));
  border-bottom: 3px solid #b999c8;
  border-left: 3px solid #b999c8;
  transform: translateX(200px);
  transition: .5s ease;
}

.show {
  transform: translateX(0px);
}

.banner-img {
  margin-top: 75px;
  height: 200px;
  width: 100%;
  background: url('./assets/smaller-cover.jpg');
  background-size: cover;
  background-position: center;
  border-bottom: 3px solid #b999c8;
  box-shadow: inset 0 -2px 18px 0px rgba(0, 0, 0, 0.5);
}

.header-logo {
  position: relative;
  top: -37.5px;
  width: 80%;
  max-width: 150px;
  min-width: 150px;
  height: 150px;
  border-radius: 50%;
  background: radial-gradient(rgb(65, 65, 65), rgb(25, 25, 25));
  box-shadow: 0 6px 18px 0px rgba(255, 255, 255, 0.15);
  display: flex;
  justify-content: center;
  align-items: center;
  margin: 0 auto;
  z-index: 5;
}

@keyframes navHover {
  0% {

  }
}

.burger {
  width: 50px;
  height: 50px;
  margin: 12px;
  position: absolute;
  top: 0;
  right: 0;
  display: flex;
  flex-direction: column;
  justify-content: space-between;

  div {
    width: 100%;
    height: 10px;
    background: $purple;
  }

  .bar1 {
    animation: bar1close .5s ease forwards;
  }

  .bar2 {
    border-top-right-radius: 0px;
    border-bottom-right-radius: 0px;
    transition: 1s ease;
  }

  .bar3 {
    animation: bar3close .5s ease forwards;
  }
}

.open {
  .bar1 {
    animation: bar1open .5s ease forwards;
  }

  .bar2 {
    border-top-right-radius: 10px;
    border-bottom-right-radius: 10px;
    transition: 1s ease;
  }

  .bar3 {
    animation: bar3open .5s ease forwards;
  }
}

@keyframes bar1open {
  0% {
    transform: rotate(0deg) translateY(0px) translateX(0);
    width: 50px;
  }
  100% {
    transform: rotate(45deg) translateY(-9px) translateX(25px);
    width: 25px;
  }
}

@keyframes bar3open {
  0% {
    transform: rotate(0deg) translateY(0px) translateX(0);
    width: 50px;
  }
  100% {
    transform: rotate(-45deg) translateY(9px) translateX(25px);
    width: 25px;
  }
}

@keyframes bar1close {
  0% {
    transform: rotate(45deg) translateY(-9px) translateX(25px);
    width: 25px;
  }
  100% {
    transform: rotate(0deg) translateY(0px) translateX(0);
    width: 50px;
  }
}

@keyframes bar3close {
  0% {
    transform: rotate(-45deg) translateY(9px) translateX(25px);
    width: 25px;
  }
  100% {
    transform: rotate(0deg) translateY(0px) translateX(0);
    width: 50px;
  }
}

@media screen and (max-width: 700px) {
  .mobile-nav {
    display: unset;
  }

  .main-nav {
    display: none;
  }
}