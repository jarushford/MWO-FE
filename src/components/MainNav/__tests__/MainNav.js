import React from 'react'
import { shallow } from 'enzyme'
import MainNav from '../MainNav'

describe('MainNav', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<MainNav />)

    expect(wrapper).toMatchSnapshot()
  })
})