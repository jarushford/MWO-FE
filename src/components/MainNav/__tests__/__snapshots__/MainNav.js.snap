// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainNav should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MainNav />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <header
          className="main-nav nav"
        >
          <NavLink
            activeClassName="active"
            aria-current="page"
            to="/tour"
          >
            tour
          </NavLink>
          <NavLink
            activeClassName="active"
            aria-current="page"
            to="/media"
          >
            media
          </NavLink>
          <Link
            replace={false}
            to="/"
          >
            <div
              className="header-logo"
            >
              <img
                alt="mw-logo"
                src="./assets/main-logo.png"
              />
            </div>
          </Link>
          <NavLink
            activeClassName="active"
            aria-current="page"
            to="/about"
          >
            about
          </NavLink>
          <NavLink
            activeClassName="active"
            aria-current="page"
            to="/contact"
          >
            contact
          </NavLink>
        </header>,
        <header
          className="mobile-nav nav"
        >
          <Link
            onClick={[Function]}
            replace={false}
            to="/"
          >
            <div
              className="header-logo"
            >
              <img
                alt="mw-logo"
                src="./assets/main-logo.png"
              />
            </div>
          </Link>
          <div
            className="burger false"
            onClick={[Function]}
          >
            <div
              className="bar1"
            />
            <div
              className="bar2"
            />
            <div
              className="bar3"
            />
          </div>
          <div
            className="dropdown-container false"
          >
            <NavLink
              activeClassName="active"
              aria-current="page"
              onClick={[Function]}
              to="/tour"
            >
              tour
            </NavLink>
            <NavLink
              activeClassName="active"
              aria-current="page"
              onClick={[Function]}
              to="/media"
            >
              media
            </NavLink>
            <NavLink
              activeClassName="active"
              aria-current="page"
              onClick={[Function]}
              to="/about"
            >
              about
            </NavLink>
            <NavLink
              activeClassName="active"
              aria-current="page"
              onClick={[Function]}
              to="/contact"
            >
              contact
            </NavLink>
          </div>
        </header>,
        <div
          className="banner-img"
        />,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <NavLink
              activeClassName="active"
              aria-current="page"
              to="/tour"
            >
              tour
            </NavLink>,
            <NavLink
              activeClassName="active"
              aria-current="page"
              to="/media"
            >
              media
            </NavLink>,
            <Link
              replace={false}
              to="/"
            >
              <div
                className="header-logo"
              >
                <img
                  alt="mw-logo"
                  src="./assets/main-logo.png"
                />
              </div>
            </Link>,
            <NavLink
              activeClassName="active"
              aria-current="page"
              to="/about"
            >
              about
            </NavLink>,
            <NavLink
              activeClassName="active"
              aria-current="page"
              to="/contact"
            >
              contact
            </NavLink>,
          ],
          "className": "main-nav nav",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "aria-current": "page",
              "children": "tour",
              "to": "/tour",
            },
            "ref": null,
            "rendered": "tour",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "aria-current": "page",
              "children": "media",
              "to": "/media",
            },
            "ref": null,
            "rendered": "media",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <div
                className="header-logo"
              >
                <img
                  alt="mw-logo"
                  src="./assets/main-logo.png"
                />
              </div>,
              "replace": false,
              "to": "/",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <img
                  alt="mw-logo"
                  src="./assets/main-logo.png"
                />,
                "className": "header-logo",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "mw-logo",
                  "src": "./assets/main-logo.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              "type": "div",
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "aria-current": "page",
              "children": "about",
              "to": "/about",
            },
            "ref": null,
            "rendered": "about",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "aria-current": "page",
              "children": "contact",
              "to": "/contact",
            },
            "ref": null,
            "rendered": "contact",
            "type": [Function],
          },
        ],
        "type": "header",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <Link
              onClick={[Function]}
              replace={false}
              to="/"
            >
              <div
                className="header-logo"
              >
                <img
                  alt="mw-logo"
                  src="./assets/main-logo.png"
                />
              </div>
            </Link>,
            <div
              className="burger false"
              onClick={[Function]}
            >
              <div
                className="bar1"
              />
              <div
                className="bar2"
              />
              <div
                className="bar3"
              />
            </div>,
            <div
              className="dropdown-container false"
            >
              <NavLink
                activeClassName="active"
                aria-current="page"
                onClick={[Function]}
                to="/tour"
              >
                tour
              </NavLink>
              <NavLink
                activeClassName="active"
                aria-current="page"
                onClick={[Function]}
                to="/media"
              >
                media
              </NavLink>
              <NavLink
                activeClassName="active"
                aria-current="page"
                onClick={[Function]}
                to="/about"
              >
                about
              </NavLink>
              <NavLink
                activeClassName="active"
                aria-current="page"
                onClick={[Function]}
                to="/contact"
              >
                contact
              </NavLink>
            </div>,
          ],
          "className": "mobile-nav nav",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <div
                className="header-logo"
              >
                <img
                  alt="mw-logo"
                  src="./assets/main-logo.png"
                />
              </div>,
              "onClick": [Function],
              "replace": false,
              "to": "/",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <img
                  alt="mw-logo"
                  src="./assets/main-logo.png"
                />,
                "className": "header-logo",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "mw-logo",
                  "src": "./assets/main-logo.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              "type": "div",
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="bar1"
                />,
                <div
                  className="bar2"
                />,
                <div
                  className="bar3"
                />,
              ],
              "className": "burger false",
              "onClick": [Function],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "bar1",
                },
                "ref": null,
                "rendered": null,
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "bar2",
                },
                "ref": null,
                "rendered": null,
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "bar3",
                },
                "ref": null,
                "rendered": null,
                "type": "div",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <NavLink
                  activeClassName="active"
                  aria-current="page"
                  onClick={[Function]}
                  to="/tour"
                >
                  tour
                </NavLink>,
                <NavLink
                  activeClassName="active"
                  aria-current="page"
                  onClick={[Function]}
                  to="/media"
                >
                  media
                </NavLink>,
                <NavLink
                  activeClassName="active"
                  aria-current="page"
                  onClick={[Function]}
                  to="/about"
                >
                  about
                </NavLink>,
                <NavLink
                  activeClassName="active"
                  aria-current="page"
                  onClick={[Function]}
                  to="/contact"
                >
                  contact
                </NavLink>,
              ],
              "className": "dropdown-container false",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "activeClassName": "active",
                  "aria-current": "page",
                  "children": "tour",
                  "onClick": [Function],
                  "to": "/tour",
                },
                "ref": null,
                "rendered": "tour",
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "activeClassName": "active",
                  "aria-current": "page",
                  "children": "media",
                  "onClick": [Function],
                  "to": "/media",
                },
                "ref": null,
                "rendered": "media",
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "activeClassName": "active",
                  "aria-current": "page",
                  "children": "about",
                  "onClick": [Function],
                  "to": "/about",
                },
                "ref": null,
                "rendered": "about",
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "activeClassName": "active",
                  "aria-current": "page",
                  "children": "contact",
                  "onClick": [Function],
                  "to": "/contact",
                },
                "ref": null,
                "rendered": "contact",
                "type": [Function],
              },
            ],
            "type": "div",
          },
        ],
        "type": "header",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "banner-img",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <header
            className="main-nav nav"
          >
            <NavLink
              activeClassName="active"
              aria-current="page"
              to="/tour"
            >
              tour
            </NavLink>
            <NavLink
              activeClassName="active"
              aria-current="page"
              to="/media"
            >
              media
            </NavLink>
            <Link
              replace={false}
              to="/"
            >
              <div
                className="header-logo"
              >
                <img
                  alt="mw-logo"
                  src="./assets/main-logo.png"
                />
              </div>
            </Link>
            <NavLink
              activeClassName="active"
              aria-current="page"
              to="/about"
            >
              about
            </NavLink>
            <NavLink
              activeClassName="active"
              aria-current="page"
              to="/contact"
            >
              contact
            </NavLink>
          </header>,
          <header
            className="mobile-nav nav"
          >
            <Link
              onClick={[Function]}
              replace={false}
              to="/"
            >
              <div
                className="header-logo"
              >
                <img
                  alt="mw-logo"
                  src="./assets/main-logo.png"
                />
              </div>
            </Link>
            <div
              className="burger false"
              onClick={[Function]}
            >
              <div
                className="bar1"
              />
              <div
                className="bar2"
              />
              <div
                className="bar3"
              />
            </div>
            <div
              className="dropdown-container false"
            >
              <NavLink
                activeClassName="active"
                aria-current="page"
                onClick={[Function]}
                to="/tour"
              >
                tour
              </NavLink>
              <NavLink
                activeClassName="active"
                aria-current="page"
                onClick={[Function]}
                to="/media"
              >
                media
              </NavLink>
              <NavLink
                activeClassName="active"
                aria-current="page"
                onClick={[Function]}
                to="/about"
              >
                about
              </NavLink>
              <NavLink
                activeClassName="active"
                aria-current="page"
                onClick={[Function]}
                to="/contact"
              >
                contact
              </NavLink>
            </div>
          </header>,
          <div
            className="banner-img"
          />,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <NavLink
                activeClassName="active"
                aria-current="page"
                to="/tour"
              >
                tour
              </NavLink>,
              <NavLink
                activeClassName="active"
                aria-current="page"
                to="/media"
              >
                media
              </NavLink>,
              <Link
                replace={false}
                to="/"
              >
                <div
                  className="header-logo"
                >
                  <img
                    alt="mw-logo"
                    src="./assets/main-logo.png"
                  />
                </div>
              </Link>,
              <NavLink
                activeClassName="active"
                aria-current="page"
                to="/about"
              >
                about
              </NavLink>,
              <NavLink
                activeClassName="active"
                aria-current="page"
                to="/contact"
              >
                contact
              </NavLink>,
            ],
            "className": "main-nav nav",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "aria-current": "page",
                "children": "tour",
                "to": "/tour",
              },
              "ref": null,
              "rendered": "tour",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "aria-current": "page",
                "children": "media",
                "to": "/media",
              },
              "ref": null,
              "rendered": "media",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": <div
                  className="header-logo"
                >
                  <img
                    alt="mw-logo"
                    src="./assets/main-logo.png"
                  />
                </div>,
                "replace": false,
                "to": "/",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <img
                    alt="mw-logo"
                    src="./assets/main-logo.png"
                  />,
                  "className": "header-logo",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "mw-logo",
                    "src": "./assets/main-logo.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                "type": "div",
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "aria-current": "page",
                "children": "about",
                "to": "/about",
              },
              "ref": null,
              "rendered": "about",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "aria-current": "page",
                "children": "contact",
                "to": "/contact",
              },
              "ref": null,
              "rendered": "contact",
              "type": [Function],
            },
          ],
          "type": "header",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <Link
                onClick={[Function]}
                replace={false}
                to="/"
              >
                <div
                  className="header-logo"
                >
                  <img
                    alt="mw-logo"
                    src="./assets/main-logo.png"
                  />
                </div>
              </Link>,
              <div
                className="burger false"
                onClick={[Function]}
              >
                <div
                  className="bar1"
                />
                <div
                  className="bar2"
                />
                <div
                  className="bar3"
                />
              </div>,
              <div
                className="dropdown-container false"
              >
                <NavLink
                  activeClassName="active"
                  aria-current="page"
                  onClick={[Function]}
                  to="/tour"
                >
                  tour
                </NavLink>
                <NavLink
                  activeClassName="active"
                  aria-current="page"
                  onClick={[Function]}
                  to="/media"
                >
                  media
                </NavLink>
                <NavLink
                  activeClassName="active"
                  aria-current="page"
                  onClick={[Function]}
                  to="/about"
                >
                  about
                </NavLink>
                <NavLink
                  activeClassName="active"
                  aria-current="page"
                  onClick={[Function]}
                  to="/contact"
                >
                  contact
                </NavLink>
              </div>,
            ],
            "className": "mobile-nav nav",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": <div
                  className="header-logo"
                >
                  <img
                    alt="mw-logo"
                    src="./assets/main-logo.png"
                  />
                </div>,
                "onClick": [Function],
                "replace": false,
                "to": "/",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <img
                    alt="mw-logo"
                    src="./assets/main-logo.png"
                  />,
                  "className": "header-logo",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "mw-logo",
                    "src": "./assets/main-logo.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                "type": "div",
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <div
                    className="bar1"
                  />,
                  <div
                    className="bar2"
                  />,
                  <div
                    className="bar3"
                  />,
                ],
                "className": "burger false",
                "onClick": [Function],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "bar1",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "bar2",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "bar3",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <NavLink
                    activeClassName="active"
                    aria-current="page"
                    onClick={[Function]}
                    to="/tour"
                  >
                    tour
                  </NavLink>,
                  <NavLink
                    activeClassName="active"
                    aria-current="page"
                    onClick={[Function]}
                    to="/media"
                  >
                    media
                  </NavLink>,
                  <NavLink
                    activeClassName="active"
                    aria-current="page"
                    onClick={[Function]}
                    to="/about"
                  >
                    about
                  </NavLink>,
                  <NavLink
                    activeClassName="active"
                    aria-current="page"
                    onClick={[Function]}
                    to="/contact"
                  >
                    contact
                  </NavLink>,
                ],
                "className": "dropdown-container false",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "activeClassName": "active",
                    "aria-current": "page",
                    "children": "tour",
                    "onClick": [Function],
                    "to": "/tour",
                  },
                  "ref": null,
                  "rendered": "tour",
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "activeClassName": "active",
                    "aria-current": "page",
                    "children": "media",
                    "onClick": [Function],
                    "to": "/media",
                  },
                  "ref": null,
                  "rendered": "media",
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "activeClassName": "active",
                    "aria-current": "page",
                    "children": "about",
                    "onClick": [Function],
                    "to": "/about",
                  },
                  "ref": null,
                  "rendered": "about",
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "activeClassName": "active",
                    "aria-current": "page",
                    "children": "contact",
                    "onClick": [Function],
                    "to": "/contact",
                  },
                  "ref": null,
                  "rendered": "contact",
                  "type": [Function],
                },
              ],
              "type": "div",
            },
          ],
          "type": "header",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "banner-img",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
